packages/itinerary-body/src/AccessLegBody/mapillary-button.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import { StreetView } from "@styled-icons/fa-solid";
+
+/**
+ * Helper method to generate bounding box from a location. Adding the WINDOW to the coordinate
+ * creates a bounding box of approximately 1 meter around the coordinate, which is likely to
+ * encompass any imagery available.
+ * @param coord     The coordinate to convert to a bounding box
+ * @returns         A bounding box 1 meter around the passed coordinate
+ */
+const generateBoundingBoxFromCoordinate = (coord: [number, number]) => {
```

Comment:
Maybe spread coord into lat and lon?

Reply:
Good call! The array was more for testing anyway. Refactored in db984ce117a40465d9a64def798af631a0dd0679